### DISCUSSION TITLE
Any features you would like to see?

### DISCUSSION THREAD

**Discussion by ssteiger** [memberrxyz/user-feedback#1] [Ideas]
Share your ideas!

**Comment by Wazzzzuzp** [comment]
What a cool app for Shopify, thank you so much.

Is it still in development?
When are you going to monetize it, how can I support you?

A couple of things if you could:

- Anychance of scaling the widget with device and having that adjustable in the widget setup app?
- Referral Settings, any chance of adding both a "Credits expire in" and a 'Minimum order value' to both Referring Referred Customers?
- The ability to change the widget Icon, a large and small icon when scaled and if you would like icon, icon text or just text.

Thanks again 

I am sure I will have more as I work through setting it up.

**Comment by ssteiger** [comment]
Hey @Wazzzzuzp 

Thank you for the feedback! We'll start working on it this week.
How did the setup go? Is there anything we can help you with?
rn we are offering a free setup workshop for all of our new customers where we integrate Store Credit deeper into your store.
You can book a call here if you are interested: https://memberr.de/en/book-product-demo-call
no must :-) we can also keep the convo here!
